### PR TITLE
GHC-118: Build Orthopedic Form

### PR DESCRIPTION
**Pull request by basrem**

### CODE REVIEW

File: configuration/concepts/orthopedics.csv [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+c6df7856-9bf2-49ff-9df1-73dd00f20101,,Nail,Nail,Clavo,Clavo,,Misc,Numeric,,,,
+1183f90c-c33a-419a-962c-1ad4e8d0ffcf,,Plate,Plate,Plato,Plato,,Misc,N/A,,,,
+c008e854-39ec-4ef8-8d8e-a7ddf6a3bfa4,,Screw,Screw,Tornillo,Tornillo,,Misc,Numeric,,,,
+89bff305-b900-473e-8970-dcae4cac4b51,,Rod,Rod,vara,vara,,Misc,N/A,,,,
+34b986d5-df71-47eb-b712-c08646533723,,Surgery,Surgery,Cirugía,Cirugía,,ConvSet,N/A,,Yes; No; Cleaning/Washout; External Fixator; Nail; Plate; Screw; Rod,,
+228e50b0-fdac-4640-84bd-9de5a534cf30,,Current Fxation,Current Fxation,Fijación actual,Fijación actual,,Misc,Text,,,,
+da181d9b-93a4-4765-8bd4-68031dbf509a,,Cast,Cast,Emitir,Emitir,,Misc,Text,,,,
+2651f961-d203-4a57-9b10-400f21c50a76,,Splint,Splint,Entablillar,Entablillar,,Misc,Text,,,,
+5d45819e-f419-496d-8fe7-59903c01ecb3,,Consevative Management,Consevative Management,Gestión conservadora,Gestión conservadora,,ConvSet,N/A,,Cast; Splint,,
+f7d0c35f-5d82-457d-a7f3-91f42720ff26,,Days in Hospital,Days in Hospital,Días en el Hospital,Días en el Hospital,,Misc,Text,,,,
```
Comment (reagan-meant) [inline configuration/concepts/orthopedics.csv]: Should this be text or numeric?

Comment (basrem) [reply configuration/concepts/orthopedics.csv]: Done making it numeric

File: configuration/concepts/orthopedics.csv [hunk 2]
```diff
@@ -0,0 +1,64 @@
+Uuid,Void/Retire,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Answers,Members,_version:1,_order:1500
+83c8f919-5bc8-4507-bf4b-85552cb6eeef,,Parts of the Body/ Bones Affected,Parts of the Body/ Bones Affected,Partes del cuerpo/huesos afectados,Partes del cuerpo/huesos afectados,,Misc,Text,,,,
```
Comment (reagan-meant) [inline configuration/concepts/orthopedics.csv]: Change to 'Parts of the Body/Bones Affected'

File: configuration/concepts/orthopedics.csv [hunk 3]
```diff
@@ -0,0 +1,64 @@
+Uuid,Void/Retire,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Answers,Members,_version:1,_order:1500
+83c8f919-5bc8-4507-bf4b-85552cb6eeef,,Parts of the Body/Bones Affected,Parts of the Body/ Bones Affected,Partes del cuerpo/huesos afectados,Partes del cuerpo/huesos afectados,,Misc,Text,,,,
+6bb7ce1f-1036-46d3-b52e-68d08c496518,,Infections,Infections,Infecciones,Infecciones,,Misc,Coded,Yes; No,,,
```
Comment (reagan-meant) [inline configuration/concepts/orthopedics.csv]: Add 'Unknown' answer

**Review by reagan-meant — CHANGES_REQUESTED** [review]

**Pull request merged**